demofile-parsing/parse_demo.js

```diff
@@ -3,71 +3,80 @@ const demofile = require("demofile");
 const path = require("path");
 const Round = require("./Round.js");
 
-const demPath = "vitality-vs-nip-dust2.dem"; //add the filepath here
-const gameStart = 13301; 
```

Comment:
yah - I think Lam is still werking on this. @tjlam thots? Seems fine to me though

Reply:
Yeah delete that boi if you need